### ISSUE TITLE
TLM-AT

### ISSUE THREAD

**Issue by myzinsky** [agra-uni-bremen/riscv-vp#3]
If I understand it correctly your VP is using the TLM-LT coding style. Have you considered also using TLM-AT for more detailed performance analysis?

**Comment by drom** [comment]
Yes, it would be interesting how your VP would bridge with AT models

**Comment by Cirromulus** [comment]
In principle it is possible to increase accuracy to AT, as long as it is still TLM.
We did not consider it yet, as it had no benefit for our usecase yet. But we are open to PRs or discussions in general.
